src/routes/expenses.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const express = require('express');
+const expensesController = require('../controllers/expenses');
+
+const router = express.Router();
+
+router.get('/', expensesController.getAll);
+router.get('/:expenseId', expensesController.get);
+
+router.post('/', express.json(), expensesController.add);
```

Comment:
No need to have this `express.json()`, as you applied it on the upper level, in `createServer.js`.
```suggestion
router.post('/', expensesController.add);
```

Same applies to rest of routes here and in user